### ISSUE TITLE
svg/extensibility/foreignObject/stacking-context-ref.html looks different on Edge than Chrome, Firefox and Safari

### ISSUE THREAD

**Issue by foolip** [w3c/web-platform-tests#11007]
Added in https://github.com/w3c/web-platform-tests/pull/8747 and renamed in https://github.com/w3c/web-platform-tests/pull/11004.

In https://wpt.fyi/svg/extensibility/foreignObject/stacking-context.html?sha=2ec5e07ed5 the test passes only in Firefox, but the reason it fails in Edge is different: the ref itself looks different from other browsers.

@chrishtr

**Comment by gsnedders** [comment]
cc/ @boggydigital @dstorey

**Comment by AmeliaBR** [comment]
The reason the reference file fails in MS Edge is because it is using the `isolation` property to force a stacking context and containing block, and Edge does not support that property. It could easily be fixed by using `position: relative; z-index: 0;` to force the stacking context instead.

That said, I'm not sure this test is backed up by any spec:
The [SVG 2 spec](https://svgwg.org/svg2-draft/embedded.html#Placement) defines that `<foreignObject>` creates an "implicitly absolutely positioned" [containing block](https://www.w3.org/TR/CSS21/visuren.html#containing-block) for the child HTML document fragment, but that doesn't automatically create a [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context).

**Comment by AmeliaBR** [comment]
Sorry, I was wrong. I found the reference part of the SVG spec in the "intent to ship" post attached to the original commit: https://svgwg.org/svg2-draft/render.html#EstablishingStackingContex (sic)

OK, so the test matches the spec, but the reference link in the test's `help` meta tag should probably be updated. *(Or I could update the SVG section on `foreignObject` to cross-reference the section on stacking contexts...)*

**Comment by foolip** [comment]
@AmeliaBR is that the same kind of isolation as is tested in any of these?:
https://wpt.fyi/results/css/compositing/mix-blend-mode/mix-blend-mode-stacking-context-creates-isolation.html
https://wpt.fyi/results/css/vendor-imports/mozilla/mozilla-central-reftests/will-change/will-change-stacking-context-isolation-1.html

Those are already Edge-only failures. Trying to figure out if the ref here should be turned into its own test...

**Comment by AmeliaBR** [comment]
@foolip 
Yes, those other tests already cover it. Nothing unusual is happening in this one that warrants its own test.
